docs/en/stack/ml/df-analytics/flightdata-regression.asciidoc

```diff
@@ -0,0 +1,367 @@
+[role="xpack"]
+[testenv="platinum"]
+[[flightdata-regression]]
+=== Predicting delays with {reganalysis} by using Kibana sample flight data
+
+
+The goal of <<dfa-regression,{reganalysis}>> is to find relationships among 
```

Comment:
This first para does not really add much and lowers the energy level into the page. Thoughts?

Reply:
@sophiec20 I tried to follow the pattern of the other example in this chapter (outlier detection) which starts with a short definition of the analysis type. I think it is necessary to emphasize in the beginning what this example is about. It seems like a natural introduction to the topic. I play with the order and the wording and fix it in my next commit.

https://www.elastic.co/guide/en/elastic-stack-overview/current/ecommerce-outliers.html